packages/common/lib/common/tools.js

```diff
@@ -3222,6 +3222,125 @@ function showDeprecatedMessage(func, logger) {
     logger(`Function "${func}" is deprecated. Please use async version of it: "${func}Async"`);
 }
 
+/**
+ * Requests the licenses from ioBroker.net
+ * @param {string} login Login for ioBroker.net
+ * @param {string} password Decoded password for ioBroker.net
+ * @returns {object[]} array of all licenses stored on iobroker.net
+ */
+function _readLicenses(login, password) {
+    axios = axios || require('axios');
+    const config = {
+        headers: { Authorization: `Basic ${Buffer.from(login + ':' + password).toString('base64')}` },
+        timeout: 4000
+    };
+
+    return axios.get(`https://iobroker.net:3001/api/v1/licenses`, config)
```

Comment:
Do you want to have the URL coded into controller? Or not hand over dynamically also ? More flexibility

Reply:
Just now we have 2 Links: diagnostics and licenses. I do not plan to change it. Anyway I can control it via DNS entries or redirect app on iobroker.net:3001